webtranslate/main.py

```diff
@@ -18,6 +18,7 @@
 from webtranslate.pages import newlanguage
 from webtranslate.pages import project_settings
 from webtranslate.pages import user_profile
+from webtranslate.pages import login
```

Comment:
feels like you should run `flake8` over your code some day :) You add an import here, but judging by the diff, you are not using it :)

Reply:
I added a comment. you have to import all pages, since they register themself.